build.gradle

```
@@ -9,7 +10,7 @@ targetCompatibility = JavaVersion.VERSION_11
 def ROBOT_MAIN_CLASS = "frc.robot.Main"
 
 // Define my targets (RoboRIO) and artifacts (deployable files)
-// This is added by GradleRIO's backing project DeployUtils.
+// This is added by GradleRIO's backing project Deployutils.
```

By mistake when replacing Utils with utils, this comment also changed, but it shouldn't.
```suggestion
// This is added by GradleRIO's backing project DeployUtils.
```